server/app/mutations/grid_services/update.rb

```diff
@@ -13,23 +13,24 @@ class Update < Mutations::Command
 
     optional do
       string :image
-      model :grid, class: Grid
-      model :stack, class: Stack
```

Comment:
I couldn't find anywhere where these were actually passed in as inputs to `GridServices::Update`.